lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
Yep, but I don't know if we have to support this. But in theory every kind of error could happen dependent on the used loader.